aws-iam-role-bless/main.tf

```diff
@@ -0,0 +1,31 @@
+resource "aws_iam_role" "client" {
+  name               = "${var.role_name}"
+  path               = "${var.iam_path}"
+  assume_role_policy = "${data.aws_iam_policy_document.client_assume_role.json}"
```

Comment:
depending on how strict we want to be, might want to use https://github.com/chanzuckerberg/cztack/tree/master/aws-iam-role-crossacct for this